kubeaudit.go

```diff
@@ -216,6 +215,26 @@ func (r *Report) Results() []Result {
 	return results
 }
 
+// Results returns the audit results for each Kubernetes resource with a minimum severity
```

Comment:
`// ResultsWithMinSeverity returns the audit results for each Kubernetes resource with a minimum severity`